rollup.config.js

```diff
@@ -1,12 +1,29 @@
+/* eslint-disable no-useless-escape */
 import { nodeResolve } from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import json from '@rollup/plugin-json';
+import replace from '@rollup/plugin-replace';
 
 export default {
-  input: 'bin.js',
+  input: {
+    bin: 'bin.js',
+    'src/worker': 'src/worker.js'
+  },
   output: {
     dir: 'output',
     format: 'cjs'
   },
-  plugins: [commonjs(), json(), nodeResolve()]
+  plugins: [
+    commonjs(),
+    json(),
+    nodeResolve({
+      exportConditions: ['node'],
+      preferBuiltins: true
+    }),
+    replace({
+      "'./src/worker.js'": 'require.resolve("./src/worker.js")',

```

Comment:
For the worker file to be included in the final binary must specify replacement to have require.resolve as `pkg` [documentation](https://www.npmjs.com/package/pkg#snapshot-filesystem) states that this will force the file to be included.